.ci/CI/src/integTestGen.jl

```diff
@@ -103,9 +104,14 @@ Generate GitLab CI job yaml for integration testing of a given package.
 function generate_job_yaml!(
     package_name::String,
     target_branch::AbstractString,
-    ci_project_dir::AbstractString,
+    dev_package_name::AbstractString,
+    dev_package_version::AbstractString,
+    dev_package_path::AbstractString,
     job_yaml::Dict,
     package_infos::AbstractDict{String,PackageInfo},
+    git_ci_tools_url::String,
+    git_ci_tools_branch::String,
```

Comment:
Is there a specific reason for these two being concrete `String`s and the other arguments being `AbstractString`s?

Reply:
No, I can change it.